examples/pytorch/text-classification/run_glue.py

```diff
@@ -516,7 +516,12 @@ def compute_metrics(p: EvalPrediction):
                             writer.write(f"{index}\t{item}\n")
 
     if training_args.push_to_hub:
-        trainer.push_to_hub()
+        kwargs = {"finetuned_from": model_args.model_name_or_path}
```

Comment:
Do you think we could directly define the `finetuned_from` model from the `Trainer`? Or is it not a good idea because some models could just have been pre-trained and not fine-tuned?

Reply:
The `Trainer` gets a model, it has no idea what the checkpoint used was.